tests/unit/client.py

```diff
@@ -295,6 +295,11 @@ def test_list_nodes_all(self):
                 u'node-06', u'node-07', u'node-08', u'node-09'
                 ]
 
+    def test_node_register(self):
+        """(successful) to node_register"""
```

Comment:
I know you copied it from other tests, but I think it reads better as "successful *call* to node_register"

Might also wanna add 2 negative tests, one with invalid subtype, and another with incorrect number of arguments.